pkg/resourcecreator/ingress/ingress.go

```diff
@@ -155,7 +173,11 @@ func nginxIngresses(source Source, cfg Config) ([]*networkingv1.Ingress, error)
 		ingressClass := util.ResolveIngressClass(rule.Host, cfg.GetGatewayMappings())
 
 		if ingressClass == nil {
-			return nil, fmt.Errorf("domain '%s' is not supported", rule.Host)
+			return nil,
+				fmt.Errorf("domain '%s' is not supported in this cluster. Your domain is '.%s', try one of these '%v'",
```

Comment:
Tror vi brukeren forstår hva `.` betyr i `'.%s'`?

Reply:
"domain foo.ansatt.nav.no is not supported in this cluster, your domain is .ansatt.nav.no, supported domains [.dev.ansatt.nav.no]"

Reply:
Fix'd: https://github.com/nais/naiserator/pull/555/commits/6598747ccc076b757338ed0c8d718ba6b595520d